api/components/actionable.schema.json

```diff
@@ -15,18 +15,23 @@
             "type": "component"
         },
         "onPressed": {
+            "description": "The function to be called when the actionable is pressed.",
```

Comment:
maybe "The listener called when the actionable is pressed." ? (same for all below)